reverse_game.js

```diff
@@ -1,64 +0,0 @@
-//Function to use verbage for async/await
-const { Console } = require('console');
```

Comment:
Sometimes when VSCode auto fills a line incorrectly it will also create an import for something that doesn't end up being used. This line looks like that may have happened and removing these can help keep your code cleaner and easier to read/understand.